libraries/helpers.rb

```diff
@@ -179,6 +179,11 @@ def json
           'ctl-command'  => nil,
           'config-file'  => nil
         },
+        'chef-compliance' => {
```

Comment:
I *think* we should use `compliance` not `chef-compliance` here. We only prefix `chef-` on `ha`, `sync`, `marketplace`, and `server` (it's kind of required on `chef` and `chef-dk`).